interface/globals.php

```diff
@@ -234,6 +235,9 @@
 $twigEnv->addFilter(new Twig_SimpleFilter('translate', function ($string) {
     return xl($string);
 }));
+$twigEnv->addFunction(new Twig_SimpleFunction('setupHeader', function ($asset = null) {
+    Header::setupHeader($asset);
+}));
 
```

Comment:
It seems like we are at a point where can break off this code into a separate openemr/twig general class that returns a $twigEnv when they are needed in applicable scripts.